ecl/hql/hqlthql.cpp

```diff
@@ -453,16 +453,23 @@ StringBuffer &HqltHql::callEclFunction(StringBuffer &s, IHqlExpression * expr, b
 
     s.append('(');
     unsigned numParameters = formals->numChildren();
+    bool first = true;
     for (unsigned idx = 0; idx < numParameters; idx++)
     {
-        if (idx)
-            s.append(", ");
-        
-        if(funcdef->isNamedSymbol())
+        if (!formals->queryChild(idx)->hasAttribute(_hidden_Atom))
```

Comment:
please add 

   expandProcessed ||

to these tests, so that when the expression tree is traced they are present.